python/lsst/daf/base/citizen.cc

```diff
@@ -15,7 +15,14 @@ PYBIND11_PLUGIN(_citizen) {
     cls.def_static("census", (int (*)(int, Citizen::memId)) &Citizen::census);
     /* It is strange to expose cout to Python, so instead make a new wrapper */
     cls.def_static("census", [](Citizen::memId id) { Citizen::census(std::cout, id); });
+    // TODO: pybind11 should logically be return_value_policy::take_ownership (dynamically allocated
+    // C++ object, passed to a local Python variable, never explicitly released), but doing so causes
+    // a segfault ("free(): invalid pointer")
```

Comment:
I suspect the information returned (at least the elements of the vector) point to data held internally to Citizen. Copy seems like the only safe option. If you agree please remove the note.

If you do keep the note then case please fix up the grammar in "pybind11 should logically be". I think replacing "be" with "use" would suffice.